docs/e2e.md

```diff
@@ -33,7 +33,13 @@ Behind the scenes it will do the following (so you don't have to do it):
 - build the API server: `npm run build`
 - start the API server: `npm run start`
 
-### 2. Frontend: Server
+### 2. Mail Server
+
+For E2E tests we need to start the [Mailpit](https://mailpit.axllent.org/) mail server to capture test emails. On a separate terminal navigate to the `opencollective-api` project and run:
+
+- `docker-compose -f docker-compose.dev.yml --profile mail up`
```

Comment:
Simpler alternative: `npm run mailpit`

Reply:
Thanks for the suggestion. I tried this and it gave me the error. At that time I didn't want to dig deep into this one. But let me see, perhaps we need some changes there. 

`
> mailpit
> ./scripts/dev/run-docker.sh compose -f docker-compose.dev.yml --profile mail up

unknown shorthand flag: 'f' in -f

Usage:  docker [OPTIONS] COMMAND [ARG...]

Run 'docker --help' for more information
`

Reply:
Proposing a fix at, https://github.com/opencollective/opencollective-api/pull/10754